### DISCUSSION TITLE
Prioritize joining channels that are currently visible to the user.

### DISCUSSION THREAD

**Discussion by mastondzn** [Chatterino/chatterino2#5194] [Ideas] [answered]
When you have bad internet and a good amount of tabs it becomes kind of annoying that you have to wait a bit for c2 to actually join the channels that you see, and that happens on each reconnect.

**Comment by pajlada** [comment]
I am hesitant about adding too much complexity for this given the EventSub chat implementation won't have this issue, however I would be open to a PR for this if it's simple enough (e.g. just a sort in a list)

**Comment by 8thony** [comment, marked as answer]
- Implemented since #5850
